app/views/publish/providers/allocations/allocation_report/_closed_state.html.erb

```diff
@@ -0,0 +1,30 @@
+<% if allocations.present? %>
+  <% unique_heading_id = rand(1...100) %>
```

Comment:
What's the purpose of this?

Reply:
Not sure, I think previously it was failing either html semantics or accessibility, it was a direct port.